thriftpy2/parser/parser.py

```diff
@@ -404,6 +410,15 @@ def set_info(self, info):
         return self.index + 1
 
 
+class CurrentFieldSeqImplicitId(object):
+    def __init__(self):
+        self.index = -1
+
+    def get_id(self):
+        self.index -= 1
+        return self.index + 1
+
```

Comment:
`CurrentFieldSeqImplicitId` should be able to be completely removed.

Reply:
Good idea. I just pushed a commit to do that.